src/locale.js

```diff
@@ -76,23 +76,40 @@ export default class Locale {
 		this.contentLanguageDirection = getLanguageDirection( this.contentLanguage );
 
 		/**
-		 * Translates the given string to the {@link #uiLanguage}. This method is also available in
+		 * Translates the given message to the {@link #uiLanguage}. This method is also available in
 		 * {@link module:core/editor/editor~Editor#t} and {@link module:ui/view~View#t}.
 		 *
-		 * The strings may contain placeholders (`%<index>`) for values which are passed as the second argument.
+		 * This method's context is statically bound to the `Locale` instance and **always should be called as a function**:
+		 *
+		 *		const t = locale.t;
+		 *		t( 'Label' );
+		 *
+		 * The message can be either a string or an object implementing the {@link module:translation-service~Message} interface.
```

Comment:
I think this link may be wrong, isn't it module:utils/translation-service~Message?